coalib/bears/Bear.py

```diff
@@ -9,6 +9,12 @@
 from coalib.settings.ConfigurationGathering import get_config_directory
 
 
+class classproperty(property):
+
+    def __get__(self, obj, type_):
+        return self.fget.__get__(None, type_)()
```

Comment:
I think you can directly convert the given function to a `classmethod`, so you don't need to write like below
```python
@classproperty
@classmethod
def ...
```